widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -331,7 +319,7 @@ public static String getWidgetsSdkVersion() {
     }
 
     /**
-     * Build time version of Glia Core SDK that is used by Glia Widgets SDK
+     * Build a time version of Glia Core SDK that is used by Glia Widgets SDK
```

Comment:
"a" is not needed here for sure.

Reply:
Could be 'Build-time'

Reply:
fixed - [cf6761a](https://github.com/salemove/android-sdk-widgets/pull/561/commits/cf6761ad6ab810bf0926f5c4b8f350825ca881f8)